src/definitions.ts

```diff
@@ -40,6 +40,15 @@ declare module '@capacitor/cli' {
         | 'scaleToFill'
         | 'scaleAspectFit'
         | 'scaleAspectFill';
+      /**
+       * Configure whether the plugin should disable/enable the possibility of taking screenshots.
+       *
+       * Only available for iOS.
+       *
+       * @default true
+       * @example true
+       */
+      disableScreenshots?: boolean;
```

Comment:
```suggestion
      /**
       * Configure whether the plugin should prevent screenshots if enabled.
       *
       * Only available for iOS.
       *
       * @default true
       * @example false
       * @since 5.2.0
       */
      preventScreenshots?: boolean;
```

Please rename this configuration option to `preventScreenshots`. This way you don't get the idea that the `enable()` method could re-enable this.

Reply:
Thank you for the correction, changes done

Reply:
Please also note my changes in the method comment.